packages/components/anchor/src/Anchor.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <component :is="wrapperComponent" :target="target" :offset="offsetTop">
+    <div class="ix-anchor-wrapper" :style="wrapperStyle">
+      <div class="ix-anchor">
+        <div class="ix-anchor-ink">
+          <span v-show="showInkInFixed" :class="inkCls" :style="{ top: `${inkTop}px` }" />
+        </div>
+        <slot></slot>
+      </div>
+    </div>
+  </component>
+</template>
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  onMounted,
+  onUnmounted,
+  watch,
+  ref,
+  nextTick,
+  provide,
+  Ref,
+  SetupContext,
+  unref,
+} from 'vue'
+import { AnchorProps, Section } from './types'
+import { PropTypes, on, off, isNil } from '@idux/cdk/utils'
+import { getOffsetTop, scrollTo, getCurrentScrollTop } from './util'
+import { getTarget } from '@idux/components/utils'
+
+export default defineComponent({
+  name: 'IxAnchor',
+  props: {
+    affix: PropTypes.bool.def(true), //固定模式, false为不浮动，状态不随页面滚动变化,默认true
+    showInkInFixed: PropTypes.bool.def(false), //固定模式是否显示小圆点,默认false
```

Comment:
```suggestion
    showInkInFixed: PropTypes.bool,
```
对于支持全局配置的 props 不能有默认值。